core/modules/system/system.install

```diff
@@ -235,6 +235,17 @@ function system_requirements($phase) {
     // Try to write the .htaccess files first, to prevent false alarms in case
     // (for example) the /tmp directory was wiped.
     file_ensure_htaccess();
+
+    foreach (array('active', 'staging') as $type) {
+      $dir = config_get_config_directory($type);
+      file_save_htaccess($dir);
+
+      $htaccess_files[$dir . '/.htaccess'] = array(
+        'title' => $t('%type config directory', array('%type' => backdrop_ucfirst($type))),
+        'directory' => $dir,
```

Comment:
Can you expand these `$dir` variables to the full spelling `$config_directory`?

Reply:
No problem, done.